packages/@react-native-windows/cli/src/runWindows/utils/deploy.ts

```diff
@@ -231,40 +231,13 @@ export async function deployToDesktop(
     windowsConfig && windowsConfig.project && windowsConfig.project.projectName
       ? windowsConfig.project.projectName
       : options.proj!;
-  const appPackageFolder = getAppPackage(options, projectName);
   const windowsStoreAppUtils = getWindowsStoreAppUtils(options);
   const appxManifestPath = getAppxManifestPath(options, projectName);
   const appxManifest = parseAppxManifest(appxManifestPath);
   const identity = appxManifest.root.children.filter(function(x) {
     return x.name === 'Identity';
   })[0];
   const appName = identity.attributes.Name;
-  const script = glob.sync(
-    path.join(appPackageFolder, 'Add-AppDevPackage.ps1'),
-  )[0];
-
-  const vsVersion = buildTools.installationVersion;
-  if (vsVersion.startsWith('16.5') || vsVersion.startsWith('16.6')) {
```

Comment:
Was this serviced for older versions, or do we need to bump min required?

Reply:
bumped to 16.7